recipes/pytest-bandit/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pytest-bandit" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "40899d30f39af3e8242cb1fd3d086317ef6f9440f3e5370c28f7212b2b15bf28"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >3
+    - setuptools
+    - pip
+  run:
+    - python
```

Also this needs an update, according to https://github.com/Wanderu/pytest-bandit/blob/master/setup.py
```suggestion
    - python >3
```